charts/newrelic-k8s-metrics-adapter/templates/_helpers.tpl

```diff
@@ -47,3 +47,31 @@ Naming helpers
 {{- define "newrelic-k8s-metrics-adapter.name.hpa-controller" -}}
 {{ include "newrelic.common.naming.truncateToDNSWithSuffix" (dict "name" (include "newrelic.common.naming.fullname" .) "suffix" "hpa-controller") }}
 {{- end -}}
+
+{{/*
+Return the custom secret name where the NR Personal API key is being stored.
+*/}}
+{{- define "newrelic-k8s-metrics-adapter.customSecretPersonalApiKeyName" -}}
+    {{- .Values.customSecretPersonalApiKeyName | default "" -}}
```

Comment:
As you are using the helpers directly, you can default them to the name of the secret in the same way you are using it in `newrelic-k8s-metrics-adapter.customSecretPersonalApiKeyKey`.

```suggestion
    {{- .Values.customSecretPersonalApiKeyName | default (include "newrelic.common.naming.fullname" .) -}}
```